### PR TITLE
188 attempt to create a add to my workouts button to ript workouts

### PR DESCRIPTION
**Pull request by nhalash**
Ript workouts can now be added by users and placed into my workouts.

### CODE REVIEW

**Review by hoangnatalie — COMMENTED** [review]
Everything worked on my end. I'd wait for Evan for the final approval.

**Review by esnowgol — APPROVED** [review]
Great Work! Appreciate your help

**Pull request merged**
